parachain/src/primitives.rs

```diff
@@ -285,3 +285,34 @@ pub struct ValidationResult {
 	/// The mark which specifies the block number up to which all inbound HRMP messages are processed.
 	pub hrmp_watermark: RelayChainBlockNumber,
 }
+
+pub type Weight = u64;
```

Comment:
Weight is a frame primitive, but we dont necessarily want all parachains to depend on frame, so redefining this here.